examples/patterns/headings/default.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+title: Headings / Default
+category: _patterns
+---
+
+<h1 class="p-heading--one">Heading one</h1>
+<h2 class="p-heading--two">Heading two</h2>
+<h3 class="p-heading--three">Heading three</h3>
+<h4 class="p-heading--four">Heading four</h4>
+<h5 class="p-heading--five">Heading five</h5>
+<h6 class="p-heading--six">Heading six</h6>
```

Comment:
Isn't this tested more thoroughly via the mixed example? We don't want me make people think they much use the classes to style headings.